app/models/user.rb

```diff
@@ -39,6 +39,11 @@ def github_client_scopes
     GitHubUser.new(github_client, uid).client_scopes
   end
 
+  # for searching, disable pagination so we can control the number of suggestions
+  def github_search_client
```

Comment:
Oops! Please ignore my comment :arrow_up: 

Looks like it doesn't work since we have conditional assignment on `@github_client`, which is assigned before the `search_client` is called.